### ISSUE TITLE
Language switching dropdown doesn’t have focus outline

### ISSUE THREAD

**Issue by kadauber** [InfosecForActivistsTeam/infosec-activists#60]
When the “English” dropdown is focused with the keyboard there is no visible indication. There should be a visible ring around the box.

**Comment by bskinner** [comment]
Fixed by #68
